ros_gz_spawn_model.launch.py seems broken on kilted

## Environment
* OS Version: Ubuntu 20.04
* Source or binary build? binary 2.1.11-1noble.20251007.045724

## Description
* Expected behavior: `ros2 launch ros_gz_sim ros_gz_spawn_model.launch.py --show-args` works
* Actual behavior:  A TypeError exception is raised

## Steps to reproduce
`ros2 launch ros_gz_sim ros_gz_spawn_model.launch.py --show-args`

## Output

```
[INFO] [launch]: All log files can be found below <...>
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught multiple exceptions when trying to load file of format [py]:
 - TypeError: 'RosGzBridge' expected 'bridge_params' to be one of [<class 'list'>], but got '<launch.substitutions.launch_configuration.LaunchConfiguration object at 0x7d04f0036f60>' of type '<class 'launch.substitutions.launch_configuration.LaunchConfiguration'>'
 - InvalidFrontendLaunchFileError: The launch file may have a syntax error, or its format is unknown
```

Here's some more details
```
  File "/opt/ros/kilted/lib/python3.12/site-packages/launch/launch_description_sources/any_launch_file_utilities.py", line 58, in get_launch_description_from_any_launch_file
    return loader(launch_file_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ros/kilted/lib/python3.12/site-packages/launch/launch_description_sources/python_launch_file_utilities.py", line 70, in get_launch_description_from_python_launch_file
    return getattr(launch_file_module, 'generate_launch_description')()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ros/kilted/share/ros_gz_sim/launch/ros_gz_spawn_model.launch.py", line 121, in generate_launch_description
    ros_gz_bridge_action = RosGzBridge(
                           ^^^^^^^^^^^^
  File "/opt/ros/kilted/lib/python3.12/site-packages/ros_gz_bridge/actions/ros_gz_bridge.py", line 97, in __init__
    ensure_argument_type(bridge_params, (list), 'bridge_params', 'RosGzBridge')
  File "/opt/ros/kilted/lib/python3.12/site-packages/launch/utilities/ensure_argument_type_impl.py", line 73, in ensure_argument_type
    raise TypeError(error_msg_template.format(
TypeError: 'RosGzBridge' expected 'bridge_params' to be one of [<class 'list'>], but got '<launch.substitutions.launch_configuration.LaunchConfiguration object at 0x7b7e8453f3e0>' of type '<class 'launch.substitutions.launch_configuration.LaunchConfiguration'>'
```

--- a/ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py
+++ b/ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py
@@ -93,6 +93,7 @@ class RosGzBridge(Action):
         self.__log_level = log_level
         self.__bridge_params = [{'config_file':  self.__config_file}]
         if bridge_params is not None:
+            bridge_params = normalize_typed_substitution(bridge_params, dict)
             # This handling of bridge_params was copied from launch_ros/actions/node.py
             ensure_argument_type(bridge_params, (list), 'bridge_params', 'RosGzBridge')
             # All elements in the list are paths to files with parameters (or substitutions that

